tests/e2e/lib.sh

```diff
@@ -26,6 +26,14 @@ deploy_stackrox() {
     sensor_wait
 }
 
+deploy_stackrox() {
+    _deploy_stackrox 2>&1 | tee "$TEST_ROOT/deployment_output.txt" || {
```

Comment:
Fine as is, but wanted to share this snippet in case you haven't seen this construct. Works in bash, zsh, sh.

```
dup_to_log() {
    local transcript="${1:-/tmp/output}"
    exec > >(tee -i "$transcript") 2>&1
}

greeting() {
    dup_to_log "/tmp/output"
    set -x
    echo "hello"
}

echo "foo"
(greeting)
echo "bar"
```

Reply:
dang. I should have tried this. `_deploy_stackrox 2>&1 | tee "$TEST_ROOT/deployment_output.txt"` clobbers the `exports` in `_deploy_stackrox` because (I guess) it creates a subshell even though not a subprocess.